INSTALL.md

```diff
@@ -87,12 +99,12 @@ for a complete development environment.
 
 To build a stable version of Singularity, check out a [release tag](https://github.com/sylabs/singularity/tags) before compiling:
 
-```
+```bash
 $ git checkout v3.0.2
```

Comment:
Should rearrange so that this is the default instruction.  We really don't want users building from master by default.